src/core/server/base.js

```diff
@@ -125,6 +125,7 @@ function baseServer(routes, createStore, { appInstanceName = appName } = {}) {
           includeSri: isDeployed,
           sriData,
           store,
+          trackingEnabled: config.get('trackingEnabled'),
```

Comment:
Yeah, it should be pretty clear and I'll I'll be checking the tracking closely once this has landed too.